IA Guide - Draft 2

Loquacity opened the pull request:
https://github.com/thegooddocsproject/planning/issues/14

Loquacity commented:
test here: http://lanabrindley.com/twinery/test.html

jaredmorgs commented:
I can't find the text area in the test site in this PR, but I noticed that if you try to view the static text version of the guide using the "click here" text, the site breaks.

>If you don’t want to do this using the interactive tool, you can **click here** to go to the static text. Just have a piece of paper handy to write down the answers to the questions as you go.

jaredmorgs commented on ia-guide/ia-cyoa.tw:
```diff
@@ -59,16 +69,16 @@ Just have a piece of paper handy to write down the answers to the questions as y
+Hi {userName}! I'm so glad you decided to join me!
 
 ## What are you writing about?
 The first thing we need to do is think about what you're writing about.
```
There is more text below this section in the section that I can't find.

>The first thing we need to do is think about what you’re writing about. Are you writing about something pretty simple, that people can use without having any experience? Like a toaster? Or are you writing about something a bit complicated, that people can start to use only after they’ve done a bit of reading, and might need to ask some questions as they go along? More like a car? Or are you writing about something really tricky? Something that needs people to have a training course, a whole lot of background knowledge, and possibly a friend to help them the whole way? Like a nuclear reactor?

It comes across as a bit of a wall of text. Can we space it out?

jaredmorgs reviewed (approved):
Couple of minor tyops found:

> The templates will help you understand how much work is involved in each of teh docs you want to write. 

s/teh/the

Observation: When I click on Further Reading I can't go back a screen to see the other link on that page.

I really like where this is going @Loquacity . The flow feels good and the prose is light enough not to be overwhelming to someone just feeling their way through documentation. 

I approve this for merge once you fix the minor tyop I found (it was the only one that jumped out at me).

Loquacity commented:
> I can't find the text area in the test site in this PR, but I noticed that if you try to view the static text version of the guide using the "click here" text, the site breaks.
> 
> > If you don’t want to do this using the interactive tool, you can **click here** to go to the static text. Just have a piece of paper handy to write down the answers to the questions as you go.

Not sure what you mean by the text area, can you give me more info?

And yes, that's just an ordinary link that doesn't yet have an endpoint, because there is no "here" to go to just yet ;)

mgan59 commented:
@Loquacity had a chance to go through the interactive tool provided in your test link.  I liked the presentation and the ability to navigate through the material.  The voice/narrative is enjoyable and I was having fun reading it.  There were a few places I felt the text could get reduced, but over-all I liked the approach and direction.  I'm 👍 for merging and we continue to iterate.

Loquacity commented:
> Couple of minor tyops found:
> 
> > The templates will help you understand how much work is involved in each of teh docs you want to write.
> 
> s/teh/the
> 

Fixed

Loquacity commented:
> Observation: When I click on Further Reading I can't go back a screen to see the other link on that page.

Added links to go back or restart.

Loquacity commented:
Updated http://lanabrindley.com/twinery/test.html :star:

viraji commented:
Firstly, I love how you have used interactive user engagement for something as tedious as the subject of Information Architecture! :)
This is really well thought out, guiding the user to just exactly what they need without them knowing it. Brilliant (smart-detection included) navigation, in that sense! 
The only thing I felt like I could do more with is, perhaps a menu at the top that could help me go back to a specific page. This is probably not an issue in the static version.

Loquacity commented:
Updated http://lanabrindley.com/twinery/test.html :star:

Loquacity commented:
Updated http://lanabrindley.com/twinery/test.html :star:

Loquacity commented:
> The only thing I felt like I could do more with is, perhaps a menu at the top that could help me go back to a specific page. This is probably not an issue in the static version.

Adding breadcrumbs is probably a little heavy for this format, and I don't think we want to encourage jumping around in the text very much, either (at least, not in the interactive version, the static version would work if you want to go back and check things). What I've done instead is add a back link to each page, and a progress bar. Hopefully that serves to give people a little more sense of where they are, while still encouraging them to go through it in a fairly linear fashion.

Loquacity commented:
Updated http://lanabrindley.com/twinery/test.html :star:

Loquacity commented:
Updated http://lanabrindley.com/twinery/test.html :star: 

Release Notes:
* Added more images
* All images are now SVG format, with artist watermark.

Loquacity commented:
Updated http://lanabrindley.com/twinery/test.html :star: 

Release Notes:
* Added more images
* All images have an updated artist watermark (https://creativecommons.org/licenses/by/3.0/)
* Critical paths table now works

flicstar commented on ia-guide/ia-cyoa.tw:
```diff
@@ -98,13 +130,20 @@ When you are thinking about your readers, make sure you are thinking about every
 There is a pretty good chance that you needed to learn how to do this thing yourself, and hopefully it wasn't all that long ago, and you can fairly easily cast your mind back to those days when you didn't know what this thing was, or how it worked.
 And if you didn't have to do this yourself, hopefully there is someone nearby who has, so go ask them questions!
 
+{embed image: 'images/car.svg', alt: 'A cheering person in a green outfit driving a red and white striped sports car, with the caption "big red car"'}
```
I really love this image. I'm not across copyright law, but I'm wondering if we're borderline with the Wiggles here?Perhaps we can change the words or remove the words from the image.  "The Big Red Car" is clearly affiliated with The Wiggles, and the song "Toot toot chugga chugga big red car" is one of their biggest hits. 
The image on this page https://wiggles.fandom.com/wiki/Toot_Toot,_Chugga_Chugga,_Big_Red_Car looks very close to the hand drawn image we're using - is it okay? Are we infringing? I'm not sure.

Loquacity replied:
I wondered this too. I think there are some small edits we're going to need to make to a few of the images before we send this live.

Loquacity commented:
Updated http://lanabrindley.com/twinery/test.html :star: 

Release Notes:
* Copyedit

Loquacity commented:
If everyone is more or less happy with the language at this point, I would like to merge this PR, and start a new one with the static version (which I'll write in markdown). We can continue to add/edit images and tweak functionality as we go.

jaredmorgs reviewed (approved):
Yep I'm happy for this to be merged. All looks good to me.
